Sources/Widgets/Core/DragHelper/index.js

```diff
@@ -0,0 +1,44 @@
+import macros from 'vtk.js/Sources/macros';
+import { add, subtract } from 'vtk.js/Sources/Common/Core/Math';
+
+function vtkWidgetDragHelper(publicAPI, model) {
+  model.classHierarchy.push('vtkWidgetDragHelper');
+
+  publicAPI.calcDelta = (dragWorldCoords) => {
+    if (dragWorldCoords?.length !== 3 || !model.lastOrigin) return [0, 0, 0];
+    return subtract(dragWorldCoords, model.lastOrigin, [0, 0, 0]);
+  };
+
+  publicAPI.calcPosition = (position, dragWorldCoords) => {
```

Comment:
you could provide an optional 3rd element that will contain the result array. (for performance reasons)